gavel/templates/admin_annotator.html

```diff
@@ -28,6 +28,10 @@
         <td>{{ annotator.secret }}</td>
       </tr>
       <tr>
+        <th>Login URL</th>
+        <td><a href="{{ login_link }}" target="_blank">{{ login_link }}</a></td>
```

Comment:
I personally wanted it to open to a new tab by default, but if you want, I'll revert to default behavior.